### PR TITLE
Issue 68

### PR DESCRIPTION
**Pull request by mindflayer**
Fix for #68.

### CODE REVIEW

File: mocket/mockhttp.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def can_handle(self, data):
         except ValueError:
             try:
                 same_entry = self == Mocket._last_entry
-                if same_entry:
+                if same_entry and decode_from_bytes(data).endswith(CRLF):
```
Comment (sjaensch) [inline mocket/mockhttp.py]: Unfortunately, this check is not accurate enough: the condition is also true for multipart requests, e.g. when doing file uploads. It fixes one of the POST request tests, but the [file upload test](https://github.com/sjaensch/aiobravado/blob/95f6c0d31020c5e18bef85bccc23bdf8903da2d8/tests/functional/request_func_test.py#L41) still fails.

Comment (mindflayer) [reply mocket/mockhttp.py]: I also have a `multipart` test, but none of them using `asyncio`. Thank you.

Comment (mindflayer) [reply mocket/mockhttp.py]: It seems better, now:
```
$ pytest tests/functional/request_func_test.py::test_file_upload_in_request
========================================================================================== test session starts ===========================================================================================
platform darwin -- Python 3.6.4, pytest-3.4.2, py-1.5.2, pluggy-0.6.0
rootdir: /Users/drizzt/repos/aiobravado, inifile: tox.ini
plugins: asyncio-0.8.0
collected 1 item

tests/functional/request_func_test.py .                                                                                                                                                            [100%]

============================================================================================ warnings summary ============================================================================================
tests/functional/request_func_test.py::test_file_upload_in_request
  /Users/drizzt/repos/aiobravado/tests/functional/conftest.py:84: RuntimeWarning: coroutine 'ClientSession.close' was never awaited
    http_client.client_session.close()

-- Docs: http://doc.pytest.org/en/latest/warnings.html
================================================================================== 1 passed, 1 warnings in 0.05 seconds ==================================================================================
```

Comment (mindflayer) [reply mocket/mockhttp.py]: Test results from `make test` are ending with a `96 passed, 17 xfailed, 2 xpassed, 46 warnings in 4.37 seconds`.
